tcms/testcases/models.py

```diff
@@ -57,6 +57,14 @@ class TestCase(TCMSActionModel):
     requirement = models.CharField(max_length=255, blank=True, null=True)
     notes = models.TextField(blank=True, null=True)
     text = models.TextField(blank=True)
+    setup_duration = models.DurationField(db_index=True, null=True, blank=True) or 0
+    testing_duration = models.DurationField(db_index=True, null=True, blank=True) or 0
+
+    @property
+    def expected_duration(self):
+        if not self.setup_duration and not self.testing_duration:
+            return 0
```

Comment:
This should be `timedelta(0)` otherwise this property will return values with 2 different data types.